src/core/components/cards/cards.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import '../cards/cards.scss';
+
+class Cards extends React.Component {
```

Comment:
Make it a summary component which iterates the gives list of cards. And also use the card CSS which is already written.